modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAxiosClientCodegen.java

```diff
@@ -126,6 +126,18 @@ public String getTypeDeclaration(Schema p) {
         }
     }
 
+    @Override
+    public Map<String, Object> postProcessOperations(Map<String, Object> objs) {
+        objs = super.postProcessOperations(objs);
+        Map<String, Object> vals = (Map<String, Object>)objs.get("operations");
+        List<CodegenOperation> operations = (List<CodegenOperation>)vals.get("operation");
```

Comment:
add comment what this does

Reply:
Addressed in 3ba25dc7ebc6a5c477a902fdbf3c7dc9ef7cbe69